lib/internal/fs/utils.js

```diff
@@ -749,6 +750,12 @@ const validateCpOptions = hideStackFrames((options) => {
   validateBoolean(options.force, 'options.force');
   validateBoolean(options.preserveTimestamps, 'options.preserveTimestamps');
   validateBoolean(options.recursive, 'options.recursive');
+  validateBoolean(options.verbatimSymlinks, 'options.verbatimSymlinks');
+  if (options.dereference === true && options.verbatimSymlinks === true) {
+    throw new ERR_INVALID_ARG_VALUE('verbatimSymlinks',
```

Comment:
In https://github.com/nodejs/node/pull/24978, @BridgeAR added the error code `ERR_INCOMPATIBLE_OPTION_PAIR`. I am wondering if that might be a better fit here.